docs/installation.rst

```diff
@@ -460,6 +460,8 @@ These platforms are built and tested for every change.
 +----------------------------------+----------------------------+---------------------+
 | CentOS Stream 8                  | 3.9                        | x86-64              |
 +----------------------------------+----------------------------+---------------------+
+| Cygwin                           | 3.8                        | x86-64              |
```

Comment:
Could move this under Windows 2019 below, like there is a separate line for MinGW?